<?xml version="1.0" encoding="UTF-8" ?>
<testsuites>
 <testsuite name="Loading" timestamp="2016-05-23T02:23:21" hostname="localhost" time="9.586" errors="0" tests="6" skipped="0" disabled="0" failures="0">
  <testcase classname="Loading" name="homepage" time="2.926" />
  <testcase classname="Loading" name="admin" time="0.446" />
  <testcase classname="Loading" name="login page" time="1.152" />
  <testcase classname="Loading" name="category page" time="2.012" />
  <testcase classname="Loading" name="product page" time="1.676" />
  <testcase classname="Loading" name="cart page" time="1.374" />
 </testsuite>
 <testsuite name="Loading" timestamp="2016-05-23T02:23:30" hostname="localhost" time="8.015" errors="0" tests="2" skipped="0" disabled="0" failures="2">
  <testcase classname="Loading" name="staging homepage" time="4.009">
   <failure type="exception" message="Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL."><![CDATA[Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
    at null._onTimeout (/www/octo-umbrella/src/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1909:23)
    at Timer.listOnTimeout (timers.js:92:15)]]>
   </failure>
  </testcase>
  <testcase classname="Loading" name="staging admin" time="4.006">
   <failure type="exception" message="Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL."><![CDATA[Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
    at null._onTimeout (/www/octo-umbrella/src/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1909:23)
    at Timer.listOnTimeout (timers.js:92:15)]]>
   </failure>
  </testcase>
 </testsuite>
</testsuites>